pkg/connect/state/state.go

```diff
@@ -156,6 +160,8 @@ type Gateway struct {
 	LastHeartbeatAtMS int64         `json:"last_heartbeat"`
 
 	Hostname string `json:"hostname"`
+
+	IPAddress net.IP `json:"ip"`
```

Comment:
Does marshaling/unmarshaling work fine with the built-in type? Do we want to store this as a string representation just in case? Had some trouble with time.Time previously which is why we're mostly using int64/string types